### PR TITLE
Add TIPS

### PR DESCRIPTION
**Pull request by bach5000**

### CODE REVIEW

File: TIPS.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+===================================
+1. Don't put spaces around "=".
+
+2. Put spaces around "[" and "]". Bash is really weird :/
+
+3. Put quotes around ALL paths that uses bash variables.
+
+4. https://www.shellcheck.net/ is your friend. Use it to your advantage!
+
+5. Use "hey [APP] getsuites" to get the properties/items on the app.
```
Comment (humdingerb) [inline TIPS.txt]: Maybe add that "hey [APP] getsuites of Window 0" gets the scripting properties of specific windows.

File: TIPS.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+6. Keep in mind that some apps require a revert to default settings before screenshot.
+   In this case, you should:
+     * Backup your settings file to the "tmp" directory included in this repo.
```
Comment (humdingerb) [inline TIPS.txt]: Use $tempDir (below as well). The "included in this repo" seems wrong.

File: TIPS.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+     * Revert settings on the app. Close app. The app's "settings" file should updated.
```
Comment (humdingerb) [inline TIPS.txt]: "... of the app."

File: TIPS.txt [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+     * Copy this settings file to the "workdir" directory. Rename to "[app].default-settings".
```
Comment (humdingerb) [inline TIPS.txt]: Use "$workfileDir".

File: TIPS.txt [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+     
+   Now, you should add some commands that copy the user's settings file to "tmp",
+   copy the default settings to the settings directory, take a screenshot of the app,
+   and restore the user's settings into the settings directory.
+   Please also remove the user settings from the "tmp" directory.
+   
+   You are welcome to look at the example scripts in the "scripts" directory :)
```
Comment (humdingerb) [inline TIPS.txt]: That's a given, I'd say.

File: TIPS.txt [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+   
+7. Add your scripts to the "scripts" directory.
```
Comment (humdingerb) [inline TIPS.txt]: That's also obvious, I'd say.

File: TIPS.txt [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+8. I recommend playing around with the "delay" function.
```
Comment (humdingerb) [inline TIPS.txt]: I think we should advise that people use "waitfor" wherever possible and only use the delay function where it isn't.

**Comment by bach5000** [comment]
I don't know if this was a good idea, but converted to Markdown, because the file still seems to be readable.

**Comment by humdingerb** [comment]
Thanks. We'll streamline the bugger over time, I think.

**Pull request merged**